orb-slot-ctrl/src/lib.rs

```diff
@@ -110,6 +118,49 @@ pub enum Slot {
     B = SLOT_B,
 }
 
+impl Slot {
+    fn variants() -> Vec<(Slot, &'static str, Vec<&'static str>)> {
+        vec![
+            (Self::A, "A", vec!["a", "0"]),
+            (Self::B, "B", vec!["b", "1"]),
+        ]
+    }
+
+    /// Retrieves a help message listing each slot variant along with its corresponding aliases.
+    #[must_use]
+    pub fn help_message() -> String {
```

Comment:
I can move it to shared trait, but this way code grew so I wasn't sure we want things to be perfect here